ament_clang_format/cmake/ament_clang_format_lint_hook.cmake

```diff
@@ -0,0 +1,14 @@
+file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
+  "*.c"
+  "*.cc"
+  "*.cpp"
+  "*.cxx"
+  "*.h"
+  "*.hh"
+  "*.hpp"
+  "*.hxx"
+)
+if(_source_files)
+  message(" - Added test 'clang_format' to check C / C++ code style")
+  ament_clang_format(clang_format)
```

Comment:
Again, `ament_clang_format(_source_files)`? At this point I'm questioning whether or not this is a mistake.